invoices/invoice_expiry_watcher.go

```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/lightningnetwork/lnd/zpay32"
 )
 
+// invoiceExpiry is a vanity interface for different invoice expiry types
```

Comment:
"different invoice expiry types" as in some use timestamp and some use block heights?

Reply:
Yeah! Becomes more apparent in to follow up PR when we have the height based expiry as well. I don't like to list the impls of an interface in the comments because that often gets outdated quickly.